doc/release-notes.md

```diff
@@ -52,553 +51,52 @@ so downgrading is not recommended.
-create a wallet directory at the specified location (containing a `wallet.dat`
-data file, a `db.log` file, and `database/log.??????????` files) instead of just
-creating a data file at the path and storing log files in the parent
-directory. This should make backing up wallets more straightforward than
-before because the specified wallet path can just be directly archived without
-having to look in the parent directory for transaction log files.
-
-For backwards compatibility, wallet paths that are names of existing data files
-in the `--walletdir` directory will continue to be accepted and interpreted the
-same as before.
-
-When Dash Core is not started with any `--wallet=<path>` options, the name of
-the default wallet returned by `getwalletinfo` and `listwallets` RPCs is
-now the empty string `""` instead of `"wallet.dat"`. If Dash Core is started
-with any `--wallet=<path>` options, there is no change in behavior, and the
-name of any wallet is just its `<path>` string.
-
-PrivateSend coin management improvements
-----------------------------------------
-A new algorithm for the creation of mixing denominations was implemented which
-should reduce the number of the smallest inputs created and give users more
-control on soft and hard caps. A much more accurate fee management algorithm was
-also implemented which should fix issues for users who have custom fees
-specified in wallet config or in times when network is under load.
-
-There is a new "PrivateSend" tab in the GUI which allows spending fully
-mixed coins only. The CoinControl feature was also improved to display coins
-based on the tab it was opened in, to protect users from accidentally spending
-mixed and non-mixed coins in the same transaction and to give better overview of
-spendable mixed coins.
-
-PrivateSend Random Round Mixing
--------------------------------
-Some potential attacks on PrivateSend assume that all inputs had been mixed
-for the same number of rounds (up to 16). While this assumption alone is not
-enough to break PrivateSend privacy, it could still provide some additional
-information for other methods like cluster analysis and help to narrow results.
-
-With Random Round Mixing, an input will be mixed to N rounds like prior. However,
-at this point, a salted hash of each input is used to pick ~50% of inputs to
-be mixed for another round. This rule is then executed again on the new inputs.
-This results in an exponential decay where if you mix a set
-of inputs, half of those inputs will be mixed for N rounds, 1/4 will be mixed N+1,
-1/8 will be mixed N+2, etc. Current implementation caps it at N+3 which results
-in mixing an average of N+0.875 rounds and sounds like a reasonable compromise
-given the privacy gains.
+Network changes
+---------------
+InstantSend locks were not relayed correctly when another node was requesting updates via `mempool`
+p2p message. Some other internal optimizations were made to the way object requests are handled.
```

Comment:
Probably not very important to mention but weren't the "Some other internal.." rather bugfixes than optimizations? (of course, bugfixes also optimize things 😆)